addons/website/static/src/components/dialog/seo.js

```diff
@@ -147,13 +148,19 @@ class Keyword extends Component {
                 lang: this.props.language,
                 keywords: this.props.keyword,
             });
-            const regex = new RegExp(WORD_SEPARATORS_REGEX + this.props.keyword + WORD_SEPARATORS_REGEX, 'gi');
+            const regex = new RegExp(
+                WORD_SEPARATORS_REGEX + escapeRegExp(this.props.keyword) + WORD_SEPARATORS_REGEX,
```

Comment:
@SergeBayet Could you define the steps to make it crash without this `escapeRegExp` in the commit message?
I'm not able to reproduce an issue without it.